lib/core/terraform_config/generator.rb

```diff
@@ -107,7 +108,7 @@ def volumeset_config_params
     def workload_config_params
       template
         .slice(:name, :description, :tags)
-        .merge(gvc: gvc, identity: workload_identity)
```

Comment:
using `identity_link` i/o `identity` since there is a case when `identity` is referenced via link in workload config and `.controlplane/templates/` does not have this identity config

Reply:
But doesn't this mean that we're excluding one case to support the other? Can we not support both cases? Or do we not need to? Or did I get it wrong?

Other than that, this PR looks good to me.

Reply:
I the use `identity` as reference then identity should be defined in `.controlplane/templates`. Having `identity_link` is more general solution because:
1. We don't require identity definition in `templates/` folder
2. Workload template (in YAML format) defines usage of `identity` via `identityLink` so it's easier to just use it i/o creating reference to identity from this link

Reply:
Got it, all good then.